src/components/Camera/PhotoPreview.js

```diff
@@ -12,13 +12,14 @@ import React, { useContext, useState } from "react";
 type Props = {
   photoUris: Array<string>,
   setPhotoUris: Function,
-  savingPhoto: boolean
+  savingPhoto: boolean,
+  deviceOrientation: string
 }
 
 const PhotoPreview = ( {
   photoUris,
   setPhotoUris,
-  savingPhoto
+  savingPhoto, deviceOrientation
```

Comment:
New line for this, please, and basically any time you see a newline-separated list of object keys.